pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -1108,6 +1108,8 @@ type Outputs struct {
 
 // WorkflowStep is a reference to a template to execute in a series of step
 type WorkflowStep struct {
+	Inline *Template `json:"inline,omitempty" protobuf:"bytes,12,opt,name=inline"`
```

Comment:
This is great! Just want to point out that this field is missing the comments.

![Screen Shot 2021-04-23 at 5 12 12 PM](https://user-images.githubusercontent.com/1311594/115930351-4c0d9280-a457-11eb-97de-b003d589dba2.png)

Reply:
already fixed 😂  please ignore